FV3/wrapper/templates/_wrapper.pyx

```diff
@@ -62,16 +63,18 @@ cdef extern:
 {% endfor %}
 
 cdef get_quantity_factory():
-    cdef int nx, ny, nz, nz_soil
+    cdef int nx, ny, nz, nz_soil, n_topo_variables
     get_centered_grid_dimensions(&nx, &ny, &nz)
     get_nz_soil_subroutine(&nz_soil)
+    get_n_orographic_variables(&n_topo_variables)
```

Comment:
Since you went with "orographic", I'd suggest naming to `n_oro_variables` or `n_orographic_variables` or even just `n_orographic`.